src/GitHub.VisualStudio/Helpers/ActiveDocumentSnapshot.cs

```diff
@@ -19,7 +19,7 @@ public class ActiveDocumentSnapshot : IActiveDocumentSnapshot
         public ActiveDocumentSnapshot([Import(typeof(SVsServiceProvider))] IServiceProvider serviceProvider)
         {
             StartLine = EndLine = -1;
-            Name = Services.Dte2?.ActiveDocument?.FullName;
+            Name = Services.Dte2?.ActiveDocument?.ProjectItem.Name;
```

Comment:
What happens when there's no project loaded? Will `ProjectItem` still exist, or will it be null? This needs to be tested by opening a file on a repo when there's no solution opened.

If `ActiveDocument` is not returning the correct file case and if `ProjectItem` can be null, then perhaps a better way of fixing this is to get the full name of the file from the filesystem to get the correct case.